### PR TITLE
feat(Chrony): Change README.md of the Chrony role

### PR DESCRIPTION
**Pull request by aldoribeirojr**
Add some information about the role on the Chrony README.md

### CODE REVIEW

File: roles/chrony/README.md [hunk 1]
```diff
@@ -1,31 +1,39 @@
-Role Name
+Chrony
 =========
```
Comment (federico-razzoli) [inline roles/chrony/README.md]: Format the headers with "#" as we did in the global README. Style must be consistent.

File: roles/chrony/README.md [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
-
-A brief description of the role goes here.
+This role installs and configures the Chrony time server (for time sincronization).
 
 Requirements
 ------------
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
+No extra packages needed.
```
Comment (federico-razzoli) [inline roles/chrony/README.md]: Mention your Ansible version (older versions may not work).

File: roles/chrony/README.md [hunk 3]
```diff
@@ -1,31 +1,39 @@
...
 
 Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
 
     - hosts: servers
+      become: true
       roles:
-         - { role: username.rolename, x: 42 }
+         - chorny
 
 License
```
Comment (federico-razzoli) [inline roles/chrony/README.md]: Replace these two final sections with "Copyright and Contacts". Copy it from the global README.
You can mention yourself at the end, I suggest to do it in this way:

Maintainer: [Aldo Junior](https://github.com/aldoribeirojr)

**Review by federico-razzoli — CHANGES_REQUESTED** [review]

File: roles/chrony/README.md [hunk 4]
```diff
@@ -1,38 +1,47 @@
-Role Name
+# Chrony
 =========
```
Comment (federico-razzoli) [inline roles/chrony/README.md]: Now you added new formatting syntax but did not delete the old formatting syntax.

**Review by federico-razzoli — CHANGES_REQUESTED** [review]

File: roles/chrony/README.md (hunk 4 above)
Comment (aldoribeirojr) [reply roles/chrony/README.md]: The changes was add on the last commit.

**Review by federico-razzoli — APPROVED** [review]

**Pull request merged**